Makefile

```diff
@@ -27,7 +27,7 @@ endif
 SHELL=/bin/bash -o pipefail -o errexit
 
 # Use the 0.0 tag for testing, it shouldn't clobber any release builds
-TAG ?= $(shell cat VERSION)
+TAG ?= $(shell cat VERSION_FILE)
```

Comment:
Please use `TAG` instead

Reply:
Hmm, sorry what do you mean?
It seems above code uses TAG.
I'm not sure how to fix. Please tell me

Reply:
Use a file named TAG

Reply:
I see! thanks.

Reply:
I fixed it.

Reply:
Please squash the commits and this lgtm

Reply:
I did it. Thanks